src/commands.c

```diff
@@ -205,8 +201,10 @@ void command_toggle_watches_proc() {
     }
 }
 
-void command_trick_proc() {
-    load_trick(fp.saved_trick);
+void command_import_save_proc() {
+    if (strcmp(fp.last_imported_save_path, "")) {
+        fp_import_file(fp.last_imported_save_path, NULL);
+    }
```

Comment:
I would personally initialize `fp.last_imported_save_path` to NULL and check against that instead of doing a string compare, but its not a huge deal either way

Reply:
That is probably better. For some reason I was thinking it'd be ugly to try and initialize in `fp_import_file()`, but now that I'm looking at it it's really not.